dxp/demystifying_aui_script_and_liferay-frontend_component_tags.md

```diff
@@ -0,0 +1,114 @@
+# Demystifying `aui:script` and `liferay-frontend:component` tags
+
+On JSP context you may notice a huge number of `aui:script` and `liferay-frontend:component` tags.
+
+## `aui:script` tag
+
+### What it does:
+
+This taglib allow users to use and require another JS module/resources inside a JSP.
```

Comment:
It's a tag, not a taglib (group of tags) ;)

Reply:
> It's a tag, not a taglib (group of tags) ;)

On my mind tag and taglib means the same. I'll review whole doc :)

Reply:
> "another" → "other" (or just remove the word entirely... less is more).

Ok mrs Wincent, thanks for the lang help here <3